packages/shared/src/lib/authentication/model/principal.ts

```diff
@@ -25,7 +27,9 @@ export type ServicePrincipal = {
 export type UserPrincipal = {
     id: ApId
     type: PrincipalType.USER
-    projectId: ProjectId
+    project: {
```

Comment:
This should be removed also from Service Principal too, check my other comment in fastify.d